modules/programs/zathura.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.programs.zathura;
+
+  formatLine = n: v:
+    let
+      formatValue = v:
+        if isBool v then (if v then "true" else "false")
+        else toString v;
+    in
+      "set ${n}\t\"${formatValue v}\"";
+in
+
+{
+  meta.maintainers = [maintainers.rprospero];
+
+  options.programs.zathura = {
+    enable = mkEnableOption ''Zathura is a highly customizable and funtional
+      document viewer focused on keyboard interaction.'';
+    options = mkOption {
+      default = null;
+      type = types.nullOr types.attrs;
+      description = ''
+      Add :set command options to zathura and make them permanent.
+      Run <code>man zathura</code> to see the full list of options
```

Comment:
When referring to manual pages can make a docbook `citerefentry`:
```xml
See
<citerefentry>
  <refentrytitle>zathurarc</refentrytitle>
  <manvolnum>5</manvolnum>
</citerefentry>
to see the full list of options.
```
I changed to `zathurarc(5)` since I guess it would contain the configuration options?

Reply:
I did not know about citerefentry.  That'll be useful in the future.